### ISSUE TITLE
Pretrained weight

### ISSUE THREAD

**Issue by bluein** [NVlabs/GCVit#7]
If click the model link, the connection is not possible due to the wrong address. can you please check it?

**Comment by avilum** [comment]
I ran into the same problem.
Looks like the links of the models refer to “add_to_model/”.

Is there any way to get the pre-trained checkpoints? 
Thanks!

**Comment by ahatamiz** [comment]
Hi @bluein and @avilum 

We have released the pre-trained weights. Please see the table for download links. 

Best,

**Comment by avilum** [comment]
Thanks a lot!

**Comment by ahatamiz** [comment]
@avilum Access permission is updated.

**Comment by vuiseng9** [comment]
@ahatamiz thanks for sharing the great work and pretrained models! however, all links are returned with "Sorry, the file you have requested does not exist. Make sure that you have the correct URL and that the file exists." Could you please check?

**Comment by ahatamiz** [comment]
Hi @vuiseng9 

Thank you for your interest in our work. We will release all model weights very soon. Please stay tuned. 

Best,

**Comment by ahatamiz** [comment]
Hi @vuiseng9 

We have updated the model and checkpoints. The new checkpoints achieve higher accuracy. 

Best,

**Comment by vuiseng9** [comment]
Wonderful! Thanks @ahatamiz! I can reproduce the accuracy score! Just tiny code tweak ```model.load_state_dict(torch.load(pretrained)['state_dict'])```, the ```state_dict``` is part of the dictionary the checkpoint.
